BLHeliBootLoad.inc

```diff
@@ -2,7 +2,13 @@
 
 XTAL					EQU	25000000
 
-BOOT_START			EQU	1C00h	; Bootloader segment address
+; Bootloader segment address
```

Comment:
Reference manual indicates that device FLASH size is 16kb for EFM8BB51F16G. 0xF000 is way beyond the end of the flash. If this address is wrapped I think it may be better to put the unwrapped one for clarity (0x3000).

Reply:
Agreed above